server/migrations/20230804034846-migrate-emoji-in-document-title.js

```diff
@@ -0,0 +1,61 @@
+"use strict";
+
+const path = require("path");
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    if (process.env.DEPLOYMENT === "hosted") {
+      return;
+    }
+
+    if (process.env.NODE_ENV === "test") {
+      return;
+    }
```

Comment:
No point running in `test` env because _all_ migrations are run as part of the test harness on a freshly created test db, so there's _no test data_ yet available.